packages/client/src/utils/readdir-recursive.ts

```diff
@@ -0,0 +1,80 @@
+// based on https://github.com/jergason/recursive-readdir
```

Comment:
Lets mention the MIT license here and also mention what was changed.

Can we send a PR upstream to fix it?

Reply:
Adding license and change info.

---

The project appears to be unmaintained. There have been no merged PRs in years. I could submit a PR, but it seems like it would go unnoticed.

It's also not necessarily a bug in the original code. They may want it to follow symlinks and not list directories as separate entries. The [readme](https://github.com/jergason/recursive-readdir#readme) calls out the behavior, but doesn't mention explicitly if that's a limitation or the intent.